swap/api/src/main/kotlin/io/buildfoundation/kaliy/swap/SwapStorage.kt

```diff
@@ -7,7 +7,7 @@ import java.nio.ByteBuffer
 interface SwapStorage {
     sealed class Data {
         sealed class Chunk {
-            class Ok(val byteBuffer: ByteBuffer): Chunk()
+            data class Ok(val byteBuffer: ByteBuffer): Chunk()
```

Comment:
ByteBuffer has equals, but it compares by capacity, actual size not content.

Reply:
Your English is getting better and better, mate!

Reply:
>ByteBuffer has equals, but it compares by capacity, actual size not content.

That's exactly why we didn't make it a `data` class, I'm still not sure if that's a good idea